koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -112,6 +112,12 @@ def parse_arguments(options, args, flatpak):
                              "Use this option to update autorebuild settings"))
     parser.add_option("--userdata",
                       help=_("JSON dictionary of user defined custom metadata"))
+    parser.add_option("--operator-csv-modifications-url",
+                      help=_("URL to JSON file with operator CSV modification"),
```

Comment:
Suggest to also mention the source pointed by this url is an alternative to the OSBS internal CSV pullspec replacement.